pkg/graph/schema.graphql

```diff
@@ -101,13 +101,13 @@ type ExistingModel {
 type TaskListSectionLockStatusChanged {
   changeType: ChangeType!
   lockStatus: TaskListSectionLockStatus!
+  role: Role!
```

Comment:
Can you clarify what the introduction of the `Role` type is here? Is this mainly to differentiate if the lock was made/unmade by an Assessment team member?

Reply:
FYI as well, https://github.com/CMSgov/mint-app/pull/176 should introduce other changes that will simplify knowing if the user is an assessment user or not.

Reply:
Great question! The purpose of the Role here is by request of FE to help them differentiate when a task was unlocked via normal behaviors (lock, unlock, context unlock) versus administrative behaviors (unlock all)